### PR TITLE
fix(ci): make the nightly tests work with evm

### PR DESCRIPTION
**Pull request by RolandSherwin**
- also increases "large file" download time from 2 mins to 3 mins inisde `merge.yml`

### CODE REVIEW

File: .github/workflows/nightly.yml [hunk 1]
```diff
@@ -26,77 +32,181 @@ jobs:
...
           # happens if we build the node manager via testnet action
           CARGO_TARGET_DIR: ${{ matrix.os == 'windows-latest' && './test-target' || '.' }}
         timeout-minutes: 15
 
-      - name: Create and fund a wallet to pay for files storage
+
+      # FIXME: do this in a generic way for localtestnets
+      - name: export default secret key
+        if: matrix.os != 'windows-latest'
+        run: echo "SECRET_KEY=0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80" >> $GITHUB_ENV
```
Comment (mazzi) [inline .github/workflows/nightly.yml:75]: This secret keys should be a repo secret maybe?

File: .github/workflows/nightly.yml [hunk 2]
```diff
@@ -26,77 +32,181 @@ jobs:
...
+        shell: bash
+      - name: Set secret key for Windows
+        if: matrix.os == 'windows-latest'
+        run: echo "SECRET_KEY=0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80" | Out-File -FilePath $env:GITHUB_ENV -Encoding utf8 -Append
```
Comment (mazzi) [inline .github/workflows/nightly.yml:79]: Same

File: .github/workflows/nightly.yml [hunk 3]
```diff
@@ -167,13 +263,12 @@ jobs:
       - name: Run logging tests
         timeout-minutes: 25
         run: cargo test --release --package sn_logging
-
+  
       - name: Run register tests
-        shell: bash
         timeout-minutes: 50
+        run: cargo test --release -package sn_registers
```
Comment (mazzi) [inline .github/workflows/nightly.yml]:
```suggestion
        run: cargo test --release --package sn_registers
```

File: .github/workflows/nightly.yml (hunk 1 above)
Comment (RolandSherwin) [reply .github/workflows/nightly.yml:75]: Hey @mazzi, this is just the local evm SECRET KEY, we don't mind exposing it.
The solution would be probably to let the CLI use it without configuring things.

Comment (RolandSherwin) [reply .github/workflows/nightly.yml:75]: It is already exposed through readme and stuff 😄

**Review by maqi — CHANGES_REQUESTED** [review]
* comment out tests (spend, token) can actually be removed ?
* can the `nightly` tests got executed somehow? or it can only be executed (to confirm they work as expected) `AFTER` the merge and wait for a day ?

**Review by maqi — APPROVED** [review]

**Pull request merged**